src/main/java/com/questhelper/QuestHelperQuest.java

```diff
@@ -224,6 +224,7 @@
 		Quest.Difficulty.ACHIEVEMENT_DIARY),
 	KARAMJA_ELITE("Elite Karamja Diary", QuestVarbits.ACHIEVEMENT_DIARY_KARAMJA_ELITE, 1, Quest.Type.ACHIEVEMENT_DIARY,
 		Quest.Difficulty.ACHIEVEMENT_DIARY),
+	FREMENNIK_EASY("Easy Fremennik Diary", QuestVarbits.ACHIEVEMENT_DIARY_FREMENNIK_EASY, 0, Quest.Type.ACHIEVEMENT_DIARY, Quest.Difficulty.ACHIEVEMENT_DIARY),
```

Comment:
A completion value of 0 means this won't run when it's not done, and will run when it is. Assuming it goes to 1 when complete, this should be 1.